sympy/core/numbers.py

```diff
@@ -505,6 +513,19 @@ class Float(Number):
     Notes
     =====
 
+    The behaviour when using the null string argument with low-precision can
```

Comment:
1.  I suppose I was worried someone will wonder why `prec=''` is not the default.

2.  Related: when I tried making `prec=''` the default, it broke simple things like `sympify('2/2.6')` which users (and our tests!) quite reasonably expect to behave like `a = 2/2.6; b = sympify(a)`.

I could just delete this whole paragraph; I don't think I have a clear point here.  So if its confusing *and* not helpful, let's just remove?